ably/types/message.py

```diff
@@ -18,8 +18,9 @@
 
 class Message(EncodeDataMixin):
     def __init__(self, name=None, data=None, client_id=None,
-                 id=None, connection_id=None, timestamp=None,
-                 encoding=''):
+                 connection_key=None, id=None, connection_id=None,
```

Comment:
It looks like you've just changed the order of the arguments in the constructor.  Are you sure that a) this constructor is not public and could break customer's code, b) that this fourth argument is not used anywhere else in the code?  Is there no way in Python to use named arguments instead to avoid this type of problem?